host.py

```diff
@@ -326,6 +358,9 @@ def copy(self, src, dst):
             self.run(f"cp {src} {dst}")
 
 
+    def get_hostname(self) -> str:
```

Comment:
can we use `hostname()` instead of `get_hostname()`. I prefer to keep the style. I know we have `bf_get_mode`, but  that's more of an exception.